app/models/katello/erratum_cve.rb

```diff
@@ -2,6 +2,14 @@ module Katello
   class ErratumCve < Katello::Model
     belongs_to :erratum, :inverse_of => :cves, :class_name => 'Katello::Erratum'
 
+    apipie :class, desc: 'A class representing Erratum CVE object' do
+      name 'Erratum CVE'
+      refs 'ErratumCve'
+      sections only: %w[all additional]
+      property :erratum, 'Erratum', desc: 'Returns Erratum object associated with this CVE'
```

Comment:
I got Uninitialized Constnat 'Erratum' here, this needs "Katello::" on the front of it